common/src/main/java/io/quarkus/ts/openshift/common/util/AwaitUtil.java

```diff
@@ -53,7 +53,9 @@ public void awaitReadiness(List<HasMetadata> resources) {
                 .forEach(it -> {
                     System.out.println(ansi().a("waiting for ").a(readableKind(it.getKind())).a(" ")
                             .fgYellow().a(it.getMetadata().getName()).reset().a(" to become ready"));
-                    await().atMost(5, TimeUnit.MINUTES).until(() -> {
+                    await().pollInterval(30, TimeUnit.SECONDS)
```

Comment:
30 seconds delay means this will slow down things a _lot_ when the resource is not immediately ready. The default is something like 100 millis or so, which might be too much, but I wouldn't go for more than 1 second.

Reply:
I agree with you. 30 seconds is too much. Changing to 1 second. Thanks!